packages/test/test-end-to-end-tests/src/test/gc/gcStateResetInSummaries.spec.ts

```diff
@@ -363,13 +379,10 @@ describeNoCompat("GC state reset in summaries", (getTestObjectProvider) => {
 
 		// Now, this summarizer has GC enabled and was loaded from a snapshot that had GC enabled. So, summary need not
 		// be regenerated from that point. However, it will receive an ack for the summary from the summarizer with GC
-		// disabled and it will refresh state from it. This should result in summary regeneration.
-		await summarizeAndValidateGCState(
-			summarizerGCEnabled2,
-			true /* shouldGCRun */,
-			true /* shouldRegenerateSummary */,
-			[newDataStore._context.id],
-		);
+		// disabled and it will close.
+		const result = summarizerGCEnabled2.summarizeOnDemand({ reason: "gcStateResetTest" });
```

Comment:
Is this summary required? Won't this summarizer close on simply catching up and waiting for CloseSummarizerDelayOverrideMs?

Reply:
Change reverted